examples/configs/user_settings_fipsv5.h

```diff
@@ -0,0 +1,147 @@
+/* user_settings_fipsv5.h
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/* should be renamed to user_settings.h for customer use
+ * generated from configure options:
+ * ./fips-check.sh linuxv5-dev keep
+ * XXX-fips-test\wolfssl\options.h
+ * ./configure --enable-fips=v5-dev
+ *
+ * Cleaned up by David Garske
+ */
+
+
+#ifndef WOLFSSL_USER_SETTINGS_H
+#define WOLFSSL_USER_SETTINGS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* FIPS Version 5.3 */
+#define HAVE_FIPS
+#define HAVE_FIPS_VERSION 5
+#define HAVE_FIPS_VERSION_MINOR 3
+
+#define HAVE_HASHDRBG /* NIST Certified DRBG - SHA256 based */
+#define HAVE_THREAD_LS
+
+/* Math */
+#define USE_FAST_MATH
+#define FP_MAX_BITS 16384
+#define SP_INT_BITS 8192
+
+/* Timing Resistance */
+#define TFM_TIMING_RESISTANT
+#define ECC_TIMING_RESISTANT
+#define WC_RSA_BLINDING
+
+/* FIPS Features */
+#define WC_RNG_SEED_CB
+#define WOLFSSL_VALIDATE_ECC_IMPORT
+#define WOLFSSL_VALIDATE_ECC_KEYGEN
+#define WOLFSSL_VALIDATE_FFC_IMPORT
+
+/* TLS Features */
+#define WOLFSSL_TLS13
+#define WOLFSSL_BASE64_ENCODE
+#define WOLFSSL_USE_ALIGN
+#define WOLFSSL_PUBLIC_MP
+#define HAVE_ENCRYPT_THEN_MAC
+#define HAVE_TLS_EXTENSIONS
+#define HAVE_SUPPORTED_CURVES
+#define HAVE_EXTENDED_MASTER
+//#define HAVE_SERVER_RENEGOTIATION_INFO
+
+#undef  WOLFSSL_WOLFSSH
```

Comment:
did you mean to keep this `#undef`?

Reply:
Yes, for some reason the configure puts it in the cflags and was getting a re-def. Only this one.